src/services/user.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * User
+ *
+ * @module User
+ */
+import http from './http';
+import session from './session';
+import storage from './storage';
+
+/**
+ * @global
+ * @typedef CLWCredits
+ * @property {Number} current user's current cl&w credits
+ * @property {Number} required user's required cl&w credits
+ */
+
+/**
+ * @global
+ * @typedef AttendedEvent
+ * @property {Object} CHDate Start time of the event
+ * @property {String} CHTermCD Term code of the event
+ * @property {Object} CHTime Time your ID was scanned
+ * @property {Number} Category_ID catagory of the event
+ * @property {String} Description Given discription of the event
+ * @property {String} Event_Name The generic name of the event
+ * @property {String} Event_Title Specific title of the event
+ * @property {String} Event_Type_Name Term code of the event
+ * @property {Object} Occurrences An array with the locations, start, and end times of future events
+ * @property {String} Organization Organization hosting the event
+ * @property {Number} Required Required chapel for the student
+*/
+
+/**
+ * Get events attended by the user
+ * @param {String} username username of the user
+ * @param {String} termCode code for the semester
+ * @return {Promise.<attendedEvents[]>} An object of all CL&W events attended by the user
```

Comment:
This needs to be changed from `attendedEvents[]` to `AttendedEvent[]`. As it is now, it will not compile correctly because the name used here does not match the `@typedef`'s name.